var/spack/repos/builtin/packages/apple-gl/package.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import sys
+
+from spack.package import *
+
+
+class AppleGl(Package):
+    """Shim package for the core OpenGL library from Apple"""
+
+    homepage = "https://developer.apple.com/library/archive/documentation/GraphicsImaging/Conceptual/OpenGL-MacProgGuide/opengl_intro/opengl_intro.html"
+
+    maintainers = ["aphecetche"]
+
+    has_code = False
+
+    version("4.1.0")
+
+    provides("gl@4.1", when=sys.platform == "darwin")
```

Comment:
The `when` option is not needed here or in the `apple-glu`.

Conceptually, a package provides the interface/API (or not).  Then `conflicts` are used if it doesn't work or apply.

Reply:
fixed. thanks.